bench-vortex/src/blob.rs

```diff
@@ -1,150 +0,0 @@
-use std::sync::Arc;
-use std::time::Duration;
-
-use async_trait::async_trait;
-use datafusion::execution::object_store::{DefaultObjectStoreRegistry, ObjectStoreRegistry};
-use futures::stream::BoxStream;
-use governor::{DefaultDirectRateLimiter, Quota};
-use object_store::path::Path;
-use object_store::{
-    GetOptions, GetResult, ListResult, MultipartUpload, ObjectMeta, ObjectStore, PutMultipartOpts,
-    PutOptions, PutPayload, PutResult, Result as OSResult,
-};
-use rand::distr::Distribution;
-use rand::rng;
-use rand_distr::LogNormal;
-use reqwest::Url;
-
-#[derive(Debug)]
-pub struct SlowObjectStore {
```

Comment:
I don't think anyone ever uses it, so might as well remove it, if we ever want to go back to using it its always available here.